influx_line_protocol/metric.py

```diff
@@ -48,17 +48,19 @@ def __str__(self):
 
         return protocol
 
-    def __escape(self, value, escape_quotes=False):
```

Comment:
I would suggest to split it up in two methods, to f.ex. __escape_value.

Reply:
Done.